### ISSUE TITLE
Neoforge 1.21.1 "is for minecraft forge or an older version of Neoforge, and cannot be loaded"

### ISSUE THREAD

**Issue by Looobay** [Numelon-Softworks/ThreatenGL#16]
I just downloaded ThreatenGL 1.3.3 for Neoforge on Modrinth and it just show this:
![Capture d'écran 2024-08-16 110825](https://github.com/user-attachments/assets/a4fa77e8-540b-44da-9642-0c15170d5db7)

**Comment by Richy-Z** [comment]
Hello, thank you for reporting this issue. Could you please try the previous version of the mod before 1.3.3? I think I may have applied updates meant for Forge to **NeoForge** because of a mislabeled GitHub issue. 😓

**Comment by Looobay** [comment]
Yeah the beta 1.3.2 for minecraft 1.21 on NeoForge works thank you !
![image](https://github.com/user-attachments/assets/28b08d87-189a-4b5c-b265-9c9006a6c513)

**Comment by Richy-Z** [comment]
Thank you for testing that previous version! I'll have to revert the changes I did to NeoForge and investigate Forge (a different issue) differently.

I'll keep this issue open for now until a new version is published on Modrinth fixes. 😊

**Comment by TheVirusNVGM** [comment]
> Я только что скачал ThreatenGL 1.3.3 для Neoforge на Modrinth, и он показывает следующее: ![Захват экрана 16 августа 2024 г. 110825](https://private-user-images.githubusercontent.com/69517054/358531931-a4fa77e8-540b-44da-9642-0c15170d5db7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM4OTMyMjMsIm5iZiI6MTcyMzg5MjkyMywicGF0aCI6Ii82OTUxNzA1NC8zNTg1MzE5MzEtYTRmYTc3ZTgtNTQwYi00NGRhLTk2NDItMGMxNTE3MGQ1ZGI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE3VDExMDg0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJlNDM1Mjc2NWVmZDU1ZjU1MjVlNzhiNDM2NDk3ZjkwM2ZlY2M5NzEzZTdkMjFhOGQ2ZDhjODFiYzk5ODU2MDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8U4X_GPLrFRWU-Au25PjiIVlKwEWE9xZPZ6ydvKtiNc)
 
you can still use the factory version together with sinytra (if you play on 1.21, you can still use sinytra version 1.21.1)

**Comment by VaporeonScripts** [comment]
i got same issue xD

**Comment by aqtoo** [comment]
Any update to this issue? As this issue is still present!!

**Comment by MarGar12** [comment]
same here, 1.21.1 neoforge 21.1.169
